nav_msgs/msg/Odometry.msg

```diff
@@ -1,7 +1,18 @@
 # This represents an estimate of a position and velocity in free space.  
 # The pose in this message should be specified in the coordinate frame given by header.frame_id.
-# The velocity in this message should be specified in the coordinate frame given by the child_frame_id
+# The velocity in this message should be specified in the coordinate frame given by the child_frame_id.
```

Comment:
I know that there is already several standards for this. Please point to existing documentation or implementations to justify these statements. I'm not entirely familiar with the current state but I think that there might be a difference between this documentation and current practices relating to the use of zeros vs negative ones.

Reply:
I took this from here: http://docs.ros.org/api/sensor_msgs/html/msg/Imu.html

except that I changed 

> If you have no estimate for one of the data elements (e.g. your IMU doesn't produce an orientation 
> estimate), please set `element 0` of the associated covariance matrix to -1

 into 

> If you have no estimate for one of the data elements, please set `_the respective diagonal_ element`
> of the associated covariance matrix to -1.

because you might know x and y of a vehicle on a road for example, but not z.